test/functional/mempool_updatefromblock.py

```diff
@@ -190,7 +197,7 @@ def test_chainlimits_exceeded(self):
 
     def run_test(self):
         # Use batch size limited by DEFAULT_ANCESTOR_LIMIT = 25 to not fire "too many unconfirmed parents" error.
```

Comment:
This comment is outdated, because `DEFAULT_ANCESTOR_LIMIT = 25` is no longer after the last commit. Not sure about the reason why it was used before, is it no longer valid?

Also, changing DEFAULT_ANCESTOR_LIMIT to 100 could be mentioned in the commit message, it also affects the other subtests.

Reply:
Kept the test setup as is, since it tests a meaningful 3 block reorg. Changed the comment because the motivation is different.